src/components/AssetManagementPanel/AssetManagementForm/FormVariants/EndorseTransferForm/EndorseTransferForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useContext, useEffect } from "react";
+import { FormState } from "../../../../../constants/FormState";
+import { ButtonSolidOrangeWhite, ButtonSolidWhiteGrey } from "../../../../UI/Button";
+import { LoaderSpinner } from "../../../../UI/Loader";
+import { AssetInformationPanel } from "../../../AssetInformationPanel";
+import { AssetManagementActions } from "../../../AssetManagementActions";
+import { AssetManagementTitle } from "../../AssetManagementTitle";
+import { OverlayContext } from "./../../../../../common/contexts/OverlayContext";
+import {
+  MessageTitle,
+  showDocumentTransferMessage,
+} from "./../../../../../components/UI/Overlay/OverlayContent/DocumentTransferMessage";
+import { EditableAssetTitle } from "./../EditableAssetTitle";
+
+interface EndorseTransferFormProps {
+  formAction: AssetManagementActions;
+  tokenId: string;
+  tokenRegistryAddress: string;
+  approvedBeneficiary: string;
+  approvedHolder: string;
+  handleEndorseTransfer: (approvedBeneficiary: string, approvedHolder: string) => void;
+  transferToNewEscrowState: string;
+  setFormActionNone: () => void;
+}
+
+export const EndorseTransferForm = ({
+  formAction,
+  tokenId,
+  tokenRegistryAddress,
+  approvedBeneficiary,
+  approvedHolder,
+  handleEndorseTransfer,
+  transferToNewEscrowState,
+  setFormActionNone,
+}: EndorseTransferFormProps) => {
+  const isPendingConfirmation = transferToNewEscrowState === FormState.PENDING_CONFIRMATION;
+  const isConfirmed = transferToNewEscrowState === FormState.CONFIRMED;
+
+  const { showOverlay } = useContext(OverlayContext);
+
+  useEffect(() => {
+    if (isConfirmed) {
+      showOverlay(
+        showDocumentTransferMessage(MessageTitle.ENDORSE_TRANSFER_SUCCESS, {
+          isSuccess: true,
+          beneficiaryAddress: approvedBeneficiary,
+          holderAddress: approvedHolder,
+        })
+      );
+      setFormActionNone();
+    }
+  }, [isConfirmed, approvedHolder, approvedBeneficiary, showOverlay, setFormActionNone]);
+
+  return (
+    <div className="row py-3">
+      <div className="col-12">
+        {!isConfirmed && (
+          <AssetManagementTitle
```

Comment:
can remove !isConfirmed le, not neccessary anymore since setFormActionNone is fired upon success

Reply:
ok